src/main/play-doc/operation/CLI.md

```diff
@@ -32,7 +32,7 @@ commands:
 ...
 ```
 
-## Packaging configuration
+## Packaging Configuration
```

Comment:
I'd use `Packaging configuration` instead. We usually write the headline words in lowercase as far as the word is not a specific term. I know that we are not fully consistent across the documentation but this is the general rule.